api-reference/10 UI Components/dxRangeSelector/1 Configuration/value/value.md

```diff
@@ -10,12 +10,194 @@ notUsedInTheme:
+            value: [50, 70]
+        })
+        // Open-ended range
+        $(#rangeSelector).dxRangeSelector({
+            value: [null, 70]
+        })
+
+    ##### Angular
+
+        <!-- tab: app.component.html -->
+        <dx-range-selector [value]="[50, 70]" ></dx-range-selector>
+        <!-- Open-ended range -->
+        <dx-range-selector [value]="[null, 70]" ></dx-range-selector>
+
+    ##### Vue
+
+        <!-- tab: App.vue -->
+        <DxRangeSelector>
+            <DxRangeSelector :value="[50, 70]" />
+        </DxRangeSelector>
+        <!-- Open-ended range -->
+        <DxRangeSelector>
+            <DxRangeSelector :value="[null, 70]" />
+        </DxRangeSelector>
+
+    ##### React
+
+        <!-- tab: App.tsx -->
+        <RangeSelector>
+            <RangeSelector value={[50, 70]} />
+        </RangeSelector>
+        <!-- Open-ended range -->
+        <RangeSelector>
+            <RangeSelector value={[null, 70]} />
+        </RangeSelector>
+
+    ---
+
+- **An object with the [startValue](/Documentation/ApiReference/UI_Components/dxRangeSelector/Configuration/value/#startValue) and [endValue](/Documentation/ApiReference/UI_Components/dxRangeSelector/Configuration/value/#endValue) fields**    
+
+    An alternative to the two-item array.
+
+    ---
+
+    ##### jQuery
+
+        <!-- tab: index.js -->
+        $(#rangeSelector).dxRangeSelector({
+            value: { startValue: 50, endValue: 70 }
+        })
+        // Open-ended range
+        $(#rangeSelector).dxRangeSelector({
+            value: { startValue: null, endValue: 70 }
+        })
+
+    ##### Angular
+
+        <!-- tab: app.component.html -->
+        <dx-range-selector [value]="{ startValue: 50, endValue: 70 }" ></dx-range-selector>
+        <!-- Open-ended range -->
+        <dx-range-selector [value]="{ startValue: null, endValue: 70 }" ></dx-range-selector>
+
+    ##### Vue
+
+    The following code snippet demonstrates **value** defined as a `<DxRangeSelector>` property:
+
+        <!-- tab: App.vue -->
+        <DxRangeSelector :value="{ startValue: null, endValue: 70 }" />
+        <!-- Open-ended range -->
+        <DxRangeSelector :value="{ startValue: null, endValue: 70 }" />
+
+    The following code snippet demonstrates **value** defined as a `<DxValue>` selector:
+
+        <DxRangeSelector>
+            <DxValue startValue="50" endValue="70" />
+        </DxRangeSelector>
+        <!-- Open-ended range -->
+        <DxRangeSelector>
+            <DxValue :startValue="null" endValue="70" />
+        </DxRangeSelector>
+
+    ##### React
+
+    The following code snippet demonstrates **value** defined as a `<RangeSelector>` property:
+
+        <!-- tab: App.tsx -->
+        <RangeSelector value={{ startValue: null, endValue: 70 }} />
+        <!-- Open-ended range -->
+        <RangeSelector value={{ startValue: null, endValue: 70 }} />
+
+    The following code snippet demonstrates value defined as a `<Value>` selector:
```

Comment:
See my comment above.  
Also, should bold font be applied to "value" (similar to the lines above)?